Gutenberg/Anatomy of a block.md

```diff
@@ -0,0 +1,55 @@
+New core editor is a powerful tool that gives you the ability to utilize the power of blocks. Blocks are a basic structure of your website.
+
+Because the new editor is in a phase of rapid development, at the time of writing this page (May 2020), it's known to make some breaking changes. This is why we opted out for creating our own block library and project library, to serve as an interface between the fast changing system.
+
+Since we use dynamic blocks, we wanted to find a way to reuse logic and components in blocks, to avoid having to write tons of boilerplate code, and to avoid breakage during development - which usually happens with standard blocks.
```

Comment:
Not sure if this is already in here, but maybe a paragraph explaining the difference between regular and dynamic blocks (or just a link to official docs) and an explanation why we use dynamic blocks (because we don't want markup stored in database + annoying block deprecation process).

Reply:
I think this is in the General chapter. Not sure about the details tho, will have to check.